docs/maven-tutorial.md

```diff
@@ -0,0 +1,218 @@
+# OpenShift + Maven Tutorial
+
+While developing your application with Java, you'll most likely be building
+many, many, many times.  In order to shorten the build times of your pods,
+dependencies can be cached in local Maven repositories.  This tutorial will step
```

Comment:
P.S. As far I understand, we're going to setup Nexus to cache downloaded artifacts somewhere inside of our infrastructure. So, when we'll run the build, Maven will download external dependencies from the local Nexus (that cached them). This means that we saved a bit of traffic and make build faster.